test/util/fs_util.h

```diff
@@ -26,6 +26,17 @@
 
 namespace gvisor {
 namespace testing {
+
+// O_LARGEFILE as defined by Linux. glibc tries to be clever by setting it to 0
+// because "it isn't needed", even though Linux can return it via F_GETFL.
+#ifdef __x86_64__
+constexpr int kOLargeFile = 00100000;
+#elif __aarch64__
```

Comment:
Shouldn't this be `#elif defined(__aarch64__)`?

Reply:
I have verified the change on both arm64 and x86 platform, and it do works! But it may seems a little weird. I will send a new patch to fix it.